andela/ah-legion-frontend
# chore (setup sass): Setup sass compilation to css

**Description**

This **PR** adds `sass` support to the project 
- It installs the package `node-sass`
- Renames `.css` files to `.scss`
- Imports and uses `index.scss` in `index.js`

<hr>
**Type of change**

- [x] chore(breaking change which adds sass support).

<hr>

**How to test this manually**
- Clone the repository `https://github.com/andela/ah-legion-frontend`
- Checkout the branch `ch-setup-sass-164046206`
- Write `sass` styles in  `.scss` files
-  Run `npm run build`
- Notice the `.scss` files are compiled to `css` in the build folder.

**Checklist:**
- [x] This follows all the guidelines for this project.

#### Related Stories

[#164046267](https://www.pivotaltracker.com/story/show/164046206)
